src/workerd/api/html-rewriter.c++

```diff
@@ -325,9 +325,11 @@ class Rewriter final: public WritableStreamSink {
   // this (buildRewriter()) modifies that vector.
   kj::Own<lol_html_HtmlRewriter> rewriter;
 
-  kj::Own<WritableStreamSink> inner;
+  // Stores data written by lol-html, which will be periodically flushed to inner.
+  kj::Vector<kj::byte> outputBuffer;
```

Comment:
Instead of an arbitrary 128MB limit per rewriter why not use ExternalMemoryAccounter to wrap the vector to correctly account for it's memory?
That will let us handle memory overallocation correctly.